app/models/users/setting.rb

```diff
@@ -4,30 +4,19 @@ class Setting < ApplicationRecord
 
     belongs_to :user
 
+    # TODO: @msarit Double-check how these suffixes have impacted the rest of the codebase
     enum editor_version: { v2: 0, v1: 1 }, _suffix: :editor
-    enum config_font: { default: 0, comic_sans: 1, monospace: 2, open_dyslexic: 3, sans_serif: 4, serif: 5 }
+    enum config_font: { default: 0, comic_sans: 1, monospace: 2, open_dyslexic: 3, sans_serif: 4, serif: 5 },
+         _suffix: :font
     enum inbox_type: { private: 0, open: 1 }, _suffix: :inbox
-    enum config_navbar: { default_navbar: 0, static_navbar: 1 }
-    enum config_theme: { default_theme: 0, minimal_light_theme: 1, night_theme: 2, pink_theme: 3,
+    enum config_navbar: { default: 0, static: 1 }, _suffix: :navbar
+    enum config_theme: { default: 0, minimal_light_theme: 1, night_theme: 2, pink_theme: 3,
                          ten_x_hacker_theme: 4 }
 
     validates :user_id, presence: true
     validates :experience_level, numericality: { less_than_or_equal_to: 10 }, allow_blank: true
     validates :feed_referential_link, inclusion: { in: [true, false] }
     validates :feed_url, length: { maximum: 500 }, allow_nil: true
     validates :inbox_guidelines, length: { maximum: 250 }, allow_nil: true
-    validate :validate_feed_url, if: :feed_url_changed?
```

Comment:
This validation was causing the `Feed::Import` specs to fail vastly and mercilessly.

The `feed_url` attribute is already validated in the User model (`user.rb:735`) and the validation is tested in the associated spec file (`user_spec.rb:280`.

So, Rhymes and I decided that, since we are still in the data-sync stage of the RFC, and the `feed_url` is copied over from the `users` table, that we could remove this validation in `users/setting.rb` for now, and re-add it when we point data updates directly to the `users_settings` table.

Reply:
@msarit I wouldn't advise removing this completely, but rather comment it out with a note. The same goes for the specs. The reason I say that is so that we can remember to put it back in else sometimes things like this slip through the cracks 😅